integreat_cms/cms/utils/content_utils.py

```diff
@@ -162,3 +163,47 @@ def fix_alt_texts(content: HtmlElement) -> None:
                 image.attrib["alt"] = media_file.alt_text
         else:
             logger.warning("Empty img tag was found.")
+
+
+def hide_anchor_tag_around_image(content: HtmlElement) -> None:
+    """
+    This function checks whether an image tag wrapped by an anchor tag has an empty alt tag, if so it hides anchor tag from screen-reader and tab-key
+
+    :param content: the content which has an anchor tag wrapped around an img tag
+    """
+
+    for anchor in content.iter("a"):
+        href = anchor.attrib.get("href")
+        # Check if the href attribute ends with the specified image extensions
+        if href and href.endswith(("png", "jpeg", "svg")):
+            # Find all img tags within the anchor tag
+            for img in anchor.iter("img"):
+                # Check if the image's alt attribute is empty
+                if not (alt_text := img.attrib.get("alt")):
```

Comment:
Nit: I think it would be a bit cleaner to swap the two cases, so that the first condition does not include a `not`:
`if (alt_text := img.attrib.get("alt")):` 
and then handle the set aria part in the else